config/webpack/react/webpack.config.dev.js

```diff
@@ -39,14 +38,22 @@ const developmentConfig = merge(baseConfig, {
           {
             loader: 'eslint-loader',
```

Comment:
So you still love it.🤪

Reply:
Only use `eslint-loader`, `fork-ts-checker-webpack-plugin` in dev mode(build react-app).
In production mode, often work with CI, there is a lint flow.